Vuologist/bumpie
# Routing header footer logic

Closes #204 , #205 

- Routing for bumpie logo and for the dashboard link
- Header and footer logic

TEST INSTRUCTIONS
1. Check header if not logged into bumpie (should match landing page in XD)
2. Footer changes color depending on pages (dashboard: yellow, landing green, about us/faq white)
3. Authenticate into bumpie header should change based on if you're logged in
